unittests/data/type_traits.hpp

```diff
@@ -421,6 +415,7 @@ namespace no{
     typedef void(*function_t)();
     typedef void (some_struct_t::*member_function_t)();
     typedef int int_t;
+    typedef const int& const_int_ref_t;
```

Comment:
Makes sense. I already stumbled on this test case, and had the same reflection, but did not dig further.

Looking at the git history, the ```typedef const int& const_int_ref_t;``` test was added a little bit later than the other tests; and the ```is_const``` part was  directly commented out so there was already a doubt on what to expect here.

+1 on that change